PRINCIPLES.md

```diff
@@ -0,0 +1,62 @@
+# GitOps Principles v0.1.0
+
+## Summary
+
+GitOps is a set of principles for operating and managing software systems.
+
+When using GitOps, the _Desired State_ of a system or subsystem is defined declaratively as versioned, immutable data, and the running system's configuration is continuously derived from this data.
+
+These principles were derived from modern software operations but are rooted in pre-existing and widely adopted best practices.
+
+## Principles
+
+1. **The principle of declarative desired state**
+
+    A system managed by GitOps must have its _Desired State_ expressed declaritively as data in a format writable and readable by both humans and machines.
+
+2. **The principle of immutable desired state versions**
+
+    _Desired State_ is stored in a way that supports versioning, immutability of versions, and retains a complete version history.
+
+3. **The principle of continuous state reconciliation**
+
+    Software agents continuously, and automatically, compare a system's _Actual State_ to its _Desired State_.
+    If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
+
+4. **The principle of operations through declaration**
+
+    The only mechanism through which a system can be intentionally operated on is through these principles.
```

Comment:
Should this be s/can be/is/?

Suggested The only mechanism through which a system is intentionally operated on is through these principles.

Reply:
Makes perfect sense, given the context of all the meetings and the general understanding of principle 4.

Reply:
Good point @jlbutler. Or maybe "the system"?

```suggestion
    The only mechanism through which the system is intentionally operated on is through these principles.
```

Since "the system" in question was already defined in principle 1 as:
>A system managed by GitOps

WDYT?

Reply:
+1 looks good to me.

Reply:
I like that wording, helps make it more specified.

Reply:
OK then, I will commit with your all of co-authorship!

Reply:
Luckily I double-check things before I do them. @jbutler will wait for your reply, since it's a proposed mod of your suggestion.

Reply:
Agreed, looks good!